src/store/actions/clients.js

```diff
@@ -0,0 +1,83 @@
+import { Toast } from '../../config/Notifications';
+import * as actionTypes from './actionTypes';
+
+import { getAllClients, createClient } from '../../services/clientService';
+
+const _fetchAllClients = () => {
+  return {
+    type: actionTypes.FETCHING_CLIENTS,
+  };
+};
+
+const _fetchAllClientsFailed = error => {
+  return {
+    type: actionTypes.FETCHING_CLIENTS_FAILURE,
+    error,
+    resultCount: 0,
+  };
+};
+
+const _receiveAllClients = (clients, count) => {
```

Comment:
just call `count` `resultCount` to simplify code :)

Also for consistency, should this not be called `_fetchAllClientsSuccess`?

Reply:
function name updated to _fetchAllClientsSuccess